### DISCUSSION TITLE
What's this pole made of?

### DISCUSSION THREAD

**Discussion by endolith** [streetcomplete/StreetComplete#4468] [Q&A] [answered]
Out of curiosity, is this information useful to someone?  I can't imagine why anyone other than the power company cares what the poles are made out of, and they probably have records already.

**Comment by Helium314** [comment, marked as answer]
See discussion in #187, and also https://github.com/streetcomplete/StreetComplete/issues/3169#issuecomment-899607510: Data is apparently used by electric grid operators, at least in France.
